app/src/main/res/values/strings.xml

```diff
@@ -1908,6 +1908,13 @@
     <string name="inactive_tabs_30_days">30 days</string>
     <!-- The amount of time until a tab in the "inactive" section of the tabs tray will be closed. See string inactive_tabs_description as well -->
     <string name="inactive_tabs_7_days">1 week</string>
+    <!-- The header of the tab auto-close message -->
```

Comment:
I think we should make these new descriptions a bit more specific (see the "7 days" description)

Reply:
These were added in July, but I'm honestly not sure these are even used. We currently don't allow the user to customize the inactive time period.

Reply:
Yeah, let's still update these descriptions. That way it's easier for the l10n folks to understand 😄